nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/remote-process-group/create-remote-process-group/create-remote-process-group.component.html

```diff
@@ -0,0 +1,115 @@
+                            [tooltipInputData]="getOptionTipData('HTTP')"
+                            [delayClose]="false">
+                            HTTP
+                        </mat-option>
+                    </mat-select>
+                </mat-form-field>
+            </div>
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>Local Network Interface</mat-label>
+                    <input matInput formControlName="localNetworkInterface" type="text" />
+                </mat-form-field>
+            </div>
+        </div>
+        <div class="flex gap-x-4">
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>HTTP Proxy Server Hostname</mat-label>
+                    <input matInput formControlName="httpProxyServerHostname" type="text" />
+                </mat-form-field>
+            </div>
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>HTTP Proxy Server Port</mat-label>
+                    <input matInput formControlName="httpProxyServerPort" type="text" />
+                </mat-form-field>
+            </div>
+        </div>
+        <div class="flex gap-x-4">
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>HTTP Proxy User</mat-label>
+                    <input matInput formControlName="httpProxyUser" type="text" />
+                </mat-form-field>
+            </div>
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>HTTP Proxy Password</mat-label>
+                    <input matInput formControlName="httpProxyPassword" type="text" />
+                </mat-form-field>
+            </div>
+        </div>
+        <div class="flex gap-x-4">
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>Communications Timeout</mat-label>
+                    <input matInput formControlName="communicationsTimeout" type="text" />
+                </mat-form-field>
+            </div>
+            <div class="w-full">
+                <mat-form-field>
+                    <mat-label>Yield Duration</mat-label>
+                    <input matInput formControlName="yieldDuration" type="text" />
+                </mat-form-field>
+            </div>
+        </div>
+    </mat-dialog-content>
+    <mat-dialog-actions align="end" *ngIf="{ value: (saving$ | async)! } as saving">
+        <button color="primary" mat-stroked-button mat-dialog-close>Cancel</button>
+        <button
+            [disabled]="!createRemoteProcessGroupForm.dirty || createRemoteProcessGroupForm.invalid || saving.value"
+            type="button"
+            color="primary"
+            (click)="createRemoteProcessGroup()"
+            mat-raised-button>
+            <span *nifiSpinner="saving.value">Add</span>
+        </button>
+    </mat-dialog-actions>
```

Comment:
Can these buttons be aligned to the bottom? In this case, this appears to be happening because the dialog content does not fill the space of the panel class that was chosen. 

![Screenshot 2024-02-09 at 10 50 07 AM](https://github.com/apache/nifi/assets/123395/b1f2073d-5cfc-422c-b7f0-fda73c6969cd)